libs/features/mas/src/price.js

```diff
@@ -88,7 +90,7 @@ export function Price({ literals, providers, settings }) {
                 method = priceAnnual;
                 break;
             default:
-                if (options.country === 'AU' && offers[0].planType === 'ABM') {
+                if (options.country === 'AU' && offers[0].planType === 'ABM' && options.displayAnnual) {
```

Comment:
`options.country === 'AU'` test should just be deferred to `options.displayAnnual`, right?

Reply:
@npeltier Yes. Right! This was originally requested for AU only but now when this feature is disabled by default, it can be used anywhere. I removed that "AU" condition.